### PR TITLE
Specify protocol in LocalCluster

### PR DESCRIPTION
**Pull request by mrocklin**
Fixes https://github.com/dask/distributed/issues/2481

cc @TomAugspurger

### CODE REVIEW

File: distributed/deploy/local.py [hunk 1]
```diff
@@ -57,6 +57,10 @@ class LocalCluster(Cluster):
     service_kwargs: Dict[str, Dict]
         Extra keywords to hand to the running services
     security : Security
+    protocol: str (optiona)
```
[the remark this answers is not in the document]
Comment (TomAugspurger) [reply distributed/deploy/local.py]: Should this be like `'tcp'` or `tcp://`?

Edit: I see that either works.

**Comment by mrocklin** [comment]
Anyway, if you get a chance to test this out I'd be curious to see if it works in your case.

**Comment by TomAugspurger** [comment]
Trying it out now. Hitting an issue in the worker creation that's probably a bug on my end.

**Comment by quasiben** [comment]
I just tested this and I'm getting:

> AttributeError: 'UCXListener' object has no attribute 'bound_address'

With the following:

> cluster = LocalCluster(protocol='ucx', ip='XX.XX.XXX.XXX', scheduler_port=13337, n_workers=2)

I think the nanny workers are trying to start up on ucx as well. 

cc @jakirkham

**Comment by mrocklin** [comment]
Can I ask for the full traceback?

Also, what happens if you don't specify an ip and scheduler_port?

**Comment by TomAugspurger** [comment]
Haven’t followed closely, but this may be blocked by https://github.com/Akshay-Venkatesh/ucx-py/issues/24#issuecomment-471528934


________________________________

**Comment by quasiben** [comment]
Same error as if you don't define ip and scheduler port:


```
/home/nfs/bzaitlen/GitRepos/distributed/distributed/bokeh/core.py:57: UserWarning: 
Port 8787 is already in use. 
Perhaps you already have a cluster running?
Hosting the diagnostics dashboard on a random port instead.
  warnings.warn('\n' + msg)
tornado.application - ERROR - Multiple exceptions in yield list
Traceback (most recent call last):
  File "/home/nfs/bzaitlen/miniconda3/envs/cudf-dev/lib/python3.7/site-packages/tornado/gen.py", line 883, in callback
    result_list.append(f.result())
  File "/home/nfs/bzaitlen/miniconda3/envs/cudf-dev/lib/python3.7/site-packages/tornado/gen.py", line 1141, in run
    yielded = self.gen.throw(*exc_info)
  File "/home/nfs/bzaitlen/GitRepos/distributed/distributed/deploy/local.py", line 251, in _start_worker
    yield w._start()
  File "/home/nfs/bzaitlen/miniconda3/envs/cudf-dev/lib/python3.7/site-packages/tornado/gen.py", line 1133, in run
    value = future.result()
  File "/home/nfs/bzaitlen/miniconda3/envs/cudf-dev/lib/python3.7/site-packages/tornado/gen.py", line 1141, in run
    yielded = self.gen.throw(*exc_info)
  File "/home/nfs/bzaitlen/GitRepos/distributed/distributed/nanny.py", line 158, in _start
    response = yield self.instantiate()
  File "/home/nfs/bzaitlen/miniconda3/envs/cudf-dev/lib/python3.7/site-packages/tornado/gen.py", line 1133, in run
    value = future.result()
  File "/home/nfs/bzaitlen/miniconda3/envs/cudf-dev/lib/python3.7/site-packages/tornado/gen.py", line 326, in wrapper
    yielded = next(result)
  File "/home/nfs/bzaitlen/GitRepos/distributed/distributed/nanny.py", line 194, in instantiate
    host = self.listener.bound_address[0]
AttributeError: 'UCXListener' object has no attribute 'bound_address'
```

**Comment by quasiben** [comment]
on the CLI the worker spins up if you set `--no-nanny`

**Pull request merged**